backend/server.js

```diff
@@ -31,15 +31,16 @@ app.use(cors({
 
 app.use(session({
   // temp secret, will be changed and moved to env file
-  secret: 'foobar',
-  resave: false,
-  saveUninitialized: false,
+  secret: 'secretCode',
+  resave: true,
+  saveUninitialized: true,
```

Comment:
I'm not sure what resave and saveUnitialized does. It would help to add a comment explaining what they do.